vault/resource_quota_lease_count.go

```diff
@@ -111,7 +123,7 @@ func quotaLeaseCountRead(d *schema.ResourceData, meta interface{}) error {
 		return nil
 	}
 
-	fields := []string{"path", "max_leases", "name"}
+	fields := []string{"path", "max_leases", "name", "inheritable"}
```

Comment:
It looks like `inheritable` was added in Vault 1.15 so we should do a separate version check for that field.